extra_data/reader.py

```diff
@@ -290,6 +290,20 @@ def __getitem__(self, item):
 
         raise TypeError("Expected data[source], data[source, key] or data[train_selection]")
 
+    def __contains__(self, item):
+        if (
+            isinstance(item, tuple) and
+            len(item) == 2 and
+            all(isinstance(e, str) for e in item)
+        ):
+            return item[1] in self._get_source_data(item[0])
```

Comment:
We still need to check the source name (`item[0]`) as well in this case - at present, it would raise `SourceNameError` instead of returning False, I think.

Reply:
Good catch, also forgot testing it.